### DISCUSSION TITLE
Keep the same axes view

### DISCUSSION THREAD

**Discussion by Ieremie** [molstar/molstar#893] [Q&A]
I am trying to compare two ESMfold predictions to a target structure. I am superimposing one of the predictions with the target, while keeping the same view when superimposing the second predictions.

This is desired as I want to plot them side by side. The 'reset axes' or 'orient axes' do not produce the same view.

**Comment by dsehnal** [comment]
You can turn on `manualReset` param for `plugin.canvas3d` and handle the camera position completely manually by using `this.plugin.managers.camera.setSnapshot`

**Reply by Ieremie** [reply]
I was looking for an option through the online interface :D

**Reply by dsehnal** [reply]
Well, reset axes always applies the same axis orientation. Maybe the problem is in the data itself?

It is also possible there is a bug, in which case it is something we should fix.

**Reply by Ieremie** [reply]
I essentially have 3 structures: 1 target and 2 predictions. When I am superimposing each prediction to the target, the final two results are not aligned the same way. This means that I have to manually rotate one of the views before taking a snapshot to have it oriented the same as the previous one.

**Reply by dsehnal** [reply]
Why dont you open the structures in the same view? You can apply "transform conformation" to it to shift it to the side:

![image](https://github.com/molstar/molstar/assets/21145776/40d6205a-099f-4130-9a23-4a3fed4bbc63)
